utils/swift_build_support/swift_build_support/products/swiftsyntax.py

```diff
@@ -81,8 +84,13 @@ def should_build(self, host_target):
 
     def build(self, host_target):
         if self.args.swiftsyntax_verify_generated_files:
+            script_path = os.path.join(self.source_dir, 'SwiftSyntaxDevUtils')
+
             build_cmd = [
-                os.path.join(self.source_dir, 'build-script.py'),
+                os.path.join(self.install_toolchain_path(host_target), "bin", "swift"),
+                'run',
+                '--package-path', script_path,
+                'swift-syntax-dev-utils',
```

Comment:
You need to set `SWIFTCI_USE_LOCAL_DEPS` here so that we don’t fetch `swift-argument-parser` from GitHub.